public/app/features/alerting/unified/components/rule-editor/VizWrapper.tsx

```diff
@@ -41,7 +37,7 @@ export const VizWrapper = ({ data, thresholds, thresholdsType }: Props) => {
                 eventBus={appEvents}
                 height={300}
                 width={width}
-                absoluteRange={timeRange}
+                timeRange={data.timeRange}
```

Comment:
@grafana/alerting-squad Please check if that won't break anything :) It looks like you're importing an Explore's component. Do you think it'd make sense to move this component somewhere to `app/core`?